run_nodejs_query.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# bash script to run nodejs query tests and save output to gs://datcom-website-periodic-testing
+
+set -e
```

Comment:
why is this file needed?

Reply:
Not needed, but followed how the rest of adversarial, screenshot and sanity does it (having a script to manually run the test). This can be used if we want to manually do a run of the nodejs query test? But I'm also ok removing it, wdyt

Reply:
This script would not work locally for autopush now? As we want to add IAP for dev, it won't work there either. 

We should have a script to trigger the cron job to run in the GKE instead...

Reply:
ah, ok, I didn't realize we were adding IAP for dev as well. I will remove this script & add a TODO for triggering the cron job to run in the GKE

Reply:
Sg

Reply:
done